components/user_settings/general/user_settings_general.test.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {shallowWithIntl} from 'tests/helpers/intl-test-helper.jsx';
+
+import UserSettingsGeneral from './user_settings_general.jsx';
+
+describe('components/user_settings/general/UserSettingsGeneral', () => {
+    const user = {
+        id: 'user_id',
+    };
+
+    const requiredProps = {
+        intl: {},
+        user,
+        updateSection: jest.fn(),
+        updateTab: jest.fn(),
+        activeSection: '',
+        prevActiveSection: '',
+        closeModal: jest.fn(),
+        collapseModal: jest.fn(),
+        actions: {
+            getMe: jest.fn(),
+            updateMe: jest.fn(),
+            sendVerificationEmail: jest.fn(),
+            setDefaultProfileImage: jest.fn(),
+            uploadProfileImage: jest.fn(),
+        },
+        maxFileSize: 1024,
+    };
+
+    test('submitUser() should have called updateMe', () => {
+        const updateMe = jest.fn(() => Promise.resolve());
```

Comment:
Resolving with `undefined` value will cause Jest test warning like [UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'data' of undefined](https://build.mattermost.com/blue/organizations/jenkins/mw%2Fmattermost-webapp/detail/PR-2361/1/pipeline/32/#step-33-log-137).  Please return with resolved value.  Empty object may work but preferably either with data or error. You may also use convenience function from Jest - `mockResolvedValue`.
```suggestion
        const updateMe = jest.fn().mockResolvedValue({data: true});
```